app/assets/stylesheets/govuk_publishing_components/components/_feedback.scss

```diff
@@ -136,60 +134,60 @@
 }
 
 .gem-c-feedback__error-message {
-  @include bold-19;
+  @include govuk-font(19, $weight: bold);
   display: block;
   padding: 4px 0 0;
-  color: $error-colour;
+  color: $govuk-error-colour;
 }
 
 .gem-c-feedback__form {
-  margin-top: $gutter-half;
-  padding: $gutter-half 0;
-  border-top: $gutter-one-third solid $govuk-blue;
+  margin-top: govuk-spacing(3);
+  padding: govuk-spacing(3) 0;
+  border-top: govuk-spacing(2) solid govuk-colour("blue");
 
-  @include media(tablet) {
-    padding: $gutter 0;
+  @include govuk-media-query($from: tablet) {
+    padding: govuk-spacing(6)0;
```

Comment:
There should be a space here between `govuk-spacing(6)` and `0` - although it seems to be applying the style correctly.

Reply:
Good spot 🔍. Updated!